WindowsServerDocs/storage/refs/refs-overview.md

```diff
@@ -99,6 +99,21 @@ Deploying ReFS as a backup target is best suited for applications and hardware t
 > [!NOTE]
 > Backup targets include the above supported configurations. Please contact application and storage array vendors for support details on Fiber Channel and iSCSI SANs. For SANs, if features such as thin provisioning, TRIM/UNMAP, or Offloaded Data Transfer (ODX) are required, NTFS must be used.
 
+### Performance considerations
+
+Pro:
+ReFS is an advanced filesystem over NTFS in many ways. It can greatly improve the performance in read / write operation scenarios. This is especially the case when comparing operations like creating Hyper-V virtual disks, snapshots or merging them. These performance benefits are not limited to Hyper-V though. 
+
+Based on real world applications this also applies for other containered data structures that are similar to the technology of VHDX, such Storage Spaces, especially when mirrored, or S2D which uses mirroring as per concept, aswell as FSLogix containers. This does also apply to Citrix PVS images, to name a 3rd party application scenario.
+
+Con:
+While **supported**, there exist scenarios where ReFS performance might not meet expectations compared to NTFS formatted volumes.
+This mostly applies to Basic Disks and volumes using ReFS without Storage Spaces / S2D, where volumes, which contain a high number of **small files and fast changing files**.
+This is by design and is caused by the needed metadata handling and scrubbing. The performance in these scenarios *can* cause slow application responsiveness or even timeouts time sensitive applications. Example: Lotus Notes ini. Other results can be unexpected low read and write speeds for random IOs compared to NTFS in the same software and hardware configuration.
+
+A typical scenario that might apply, based on the underlying hardware configuration, are typically Windows User Profiles, Temporary Folders of browsers in high-density or multi-user enviroments, such as Microsoft Remote Desktop Host (RDSH), Windows Virtual Desktop (WVD), or 3rd party applications like Citrix User Profile Management (UPM).
```

Comment:
```suggestion
A typical scenario that may apply, based on the underlying hardware configuration, are typically Windows User Profiles, Temporary Folders of browsers in high-density or multi-user enviroments, such as Microsoft Remote Desktop Host (RDSH), Windows Virtual Desktop (WVD), or 3rd party applications like Citrix User Profile Management (UPM).
```
Again, same reason as above, use "may" instead of "might" whenever there is at least just as high probability for something to happen, as for it not to happen.